lib/shopify-cli/js_system.rb

```diff
@@ -81,18 +81,30 @@ def yarn?
     #
     #   ShopifyCli::JsSystem.new(ctx: ctx).call(yarn: ['install', '--silent'], npm: ['install', '--no-audit'])
     #
-    def call(yarn:, npm:)
-      yarn? ? call_command(yarn, YARN_CORE_COMMAND) : call_command(npm, NPM_CORE_COMMAND)
+    def call(yarn:, npm:, capture_response: false)
```

Comment:
There are some comments above this method that should be updated since we have added a new parameter.